bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -8,164 +8,311 @@
-        {{ release.text|markdown|safe }}
-      </p>
+    {% if release.version %}
+      <div class="latest-release" >
+        <div class="container">
+          {% block notes_heading_secondary %}
+          <div class="version">
+            <h3>{{ _('{version}')|f(version=release.version) }}</h3>
+            <h4>{{ release.product }} {{ release.channel }}</h4>
+            <p>{{ _('{date}')|f(date=release.release_date|l10n_format_date) }}</p>
+          </div>
+          <div class="description">
+              <h2>{{ _('Version {version}, first offered to {channel} channel users on {date}')|f(channel=release.channel, date=release.release_date|l10n_format_date, version=release.version) }}</h2>
+              <p>{{ release.text|markdown|safe }}</p>
+          </div>
+          {% endblock %}
+        </div>
+      </div>
+    {% endif %}
     </header>
   {% endblock %}
 
-    <div class="main-column">
-      {% if new_features %}
-      <section class="notes-section" id="new-features">
-        <h3>{{ _('What’s New') }}</h3>
-        <ul class="section-items tagged">
-          {% for note in new_features %}
-          <li {% if not note.tag %}class="untagged"{% endif %} id="note-{{ note.id }}">
-            {% if note.tag %}
-              <b class="tag tag-{{ note.tag.lower() }}">{{ note.tag }}</b>
+    <section class="notes-section">
+      <div class="features" id="new-features">
+        <div class="container">
+        {% if new_features %}
+          {% for note in new_features if note.tag == "New" %}
+            {% if loop.first %}
+            <div class="section-wrapper">
+              <p id="{{ note.tag }}">{{ note.tag }}</p>
+              <ul class="section-items">
+              {% endif %}
+                <li id="note-{{ note.id }}">
+                  <p>{{ note.note|markdown|safe }}</p>
+                </li>
+                {% if loop.last %}
+              </ul>
+            </div>
             {% endif %}
-              {{ note.note|markdown|safe }}
-          </li>
           {% endfor %}
-        </ul>
-      </section>
-      {% endif %}
-      {% if known_issues %}
-      <section class="notes-section" id="known-issues">
-        <h3>{{ _('Known Issues') }}</h3>
-        <ul class="section-items tagged">
-          {% for note in known_issues %}
-            <li id="note-{{ note.id }}">
-                <b class="tag tag-unresolved">{{ _('unresolved') }}</b>
-                {{ note.note|markdown|safe }}
-            {% if note.fixed_in_release %}
-              <p class="note">
-                <a href="{{ releasenotes_url(note.fixed_in_release) }}">
-                  {{ _('Resolved in v{version_number}')|f(version_number=note.fixed_in_release.version) }}
-                </a>
-              </p>
+
+          {% for note in new_features if note.tag == "Fixed" %}
+            {% if loop.first %}
+            <div class="section-wrapper">
+              <p id="{{ note.tag }}">{{ note.tag }}</p>
```

Comment:
Same thing here; heading with a lowercase ID. Though come to think of it, if the only purpose for these IDs is to identify the section so we can link to it, it should be on the section's container and not the heading.